oidc-samples/OIDC-SDK/org.wso2.identity.sdk.oidc/obj/Release/netstandard2.0/org.wso2.identity.sdk.oidc.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,76 @@
+C:\Users\wso2\source\repos\Project - IAM\org.wso2.identity.sdk.oidc\org.wso2.identity.sdk.oidc\bin\Release\netstandard2.0\org.wso2.identity.sdk.oidc.deps.json
```

Comment:
This file still contains absolute paths. According to [1], we don't need to commit these files.

[1] https://stackoverflow.com/questions/559045/what-are-all-these-filelistabsolute-txt-files-for

Reply:
Resolved.